### PR TITLE
Bump eks-anywhere-packages version

### PR DESCRIPTION
**Pull request by abhay-krishna**
Bump Git tag for eks-anywhere-packages project.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Common.mk [hunk 1]
```diff
@@ -23,6 +23,7 @@ SHELL=$(DEFAULT_SHELL)
 AWS_REGION?=us-west-2
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 ARTIFACTS_BUCKET?=s3://my-s3-bucket
+DEPENDENCIES_BUCKET?=$(ARTIFACTS_BUCKET)
```
[the remark this answers is not in the document]
Comment (abhay-krishna) [reply Common.mk:26]: I was just testing cross-account artifact dependencies fetching, not gonna be merging this change.

**Comment by d8660091** [comment]
We need to update curated packages bundle yaml after this PR, right?

**Comment by abhay-krishna** [comment]
> We need to update curated packages bundle yaml after this PR, right?

Exactly

**Review by jaxesn — APPROVED** [review]

**Review by d8660091 — APPROVED** [review]

**Pull request merged**